reflex/constants/route.py

```diff
@@ -50,9 +50,9 @@ class DefaultPage(SimpleNamespace):
     """Default page constants."""
 
     # The default title to show for Reflex apps.
-    TITLE = "Reflex App"
+    TITLE = "{} : {}"
```

Comment:
I think using `|` or `-` instead of `:` is more standard - but I like this approach!

Reply:
I'm not sure this is the case I was looking at this SEO article and most use":"https://blog.hubspot.com/blog/tabid/6307/bid/5984/spilling-seo-juice-3-dos-and-don-ts-for-writing-great-page-titles.aspx

Reply:
If we use a colon let's remove the space before, but I personally prefer the `|`

Reply:
Ok just do the |